### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by warber**

### CODE REVIEW

File: releasenotes/releasenotes_V0.9.0.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Release Notes develop
+# Release Notes 0.9.0
```
Comment (agrimmer) [inline releasenotes/releasenotes_V0.9.0.md:2]: Why are the release notes empty?

Comment (warber) [reply releasenotes/releasenotes_V0.9.0.md:2]: they are not

Comment (warber) [reply releasenotes/releasenotes_V0.9.0.md:2]: https://github.com/keptn-contrib/dynatrace-sli-service/blob/cc4b54255c4f03dd3ddf426759405587d507cae6/releasenotes/releasenotes_V0.9.0.md

Comment (agrimmer) [reply releasenotes/releasenotes_V0.9.0.md:2]: Sorry, I was too fast.

**Review by agrimmer — DISMISSED** [review]
lgtm

File: deploy/service.yaml [hunk 2]
```diff
@@ -51,7 +51,7 @@ spec:
       serviceAccountName: keptn-dynatrace-sli-service
       containers:
         - name: dynatrace-sli-service
-          image: keptncontrib/dynatrace-sli-service:0.8.0
+          image: keptncontrib/dynatrace-sli-service:0.9.0
```
Comment (agrimmer) [inline deploy/service.yaml:54]: Should we use `distributor:0.8.1` ?

Comment (warber) [reply deploy/service.yaml:54]: clarified with @bacherfl - not neccessary

**Review by agrimmer — APPROVED** [review]
lgtm

**Pull request merged**